.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,40 +1,40 @@
 ---
-name: Bug report
-about: Crashes, inaccurate emulation, nitpicks, or regressions
-title: '(issue title -- summarise the summary)'
+name: Bug Report
+about: Report broken or incorrect behavior
+title: '(Issue title -- summarise issue content)'
 labels: ''
 assignees: ''
 
 ---
 
-[//]: # "This issue description supports Markdown syntax (this is what comments look like). There's a cheatsheet here: https://guides.github.com/features/mastering-markdown/"
-[//]: # "You can leave these comments here or delete them. Also, please remember to search for similar issues before writing anything, including in closed issues!"
-
-## Summary
-[//]: # "Briefly describe what's broken. Include relevant details: loaded core, loaded rom's hash, open tools, running scripts... You can embed a screenshot if it's easier to show the bug, but if you need more than one please put them at the end."
-Games run faster than I can push buttons whenever my cat sits on the left side of my keyboard.
-
-## Repro
-[//]: # "If you can't figure out the list of steps, delete this section and put 'heisenbug' in the summary somewhere. If a Lua script can cause the bug, you can embed that instead (as simple as possible please)."
-1. first step
-2. second step
-3. et cetera
-
-## Output
-[//]: # "Paste the contents of the error dialog if there is one (Ctrl+C works even if it looks like it doesn't), or paste the output from the Lua Console, or delete this section."
-```
-o noes
-  at BizHawk.Client.EmuHawk.HawkBiz.RunWithoutCrashing()
-```
-
-## Host env.
-[//]: # "List the computers you've seen the bug happen on. You'll probably need to provide more detailed specs for graphical/driver issues. Here are some examples:"
-- BizHawk 2.3.2; Win10 Pro 1903; Intel/AMD
-- BizHawk 2.3.2; Win10 Home 1809; AMD/NVIDIA
-- BizHawk 2.3.2; Fedora 30; AMD/AMD
-- BizHawk 2.3.2; macOS 10.15; Intel/AMD
-- BizHawk 1.13.2; Win7 SP1; Intel/NVIDIA
-
-[//]: # "If you know the bug does *not* occur on a certain version of EmuHawk, or on a certain OS, please list those too and add 'unaffected'."
-
-[//]: # "(screenshots, if applicable)"
+### Summary
+
+<!-- A summary of your bug report. -->
+
+### Reproduction Steps
+
+<!-- What you were doing that caused the issue. Consider adding screenshots to help others reproduce the bug. -->
+<!-- Include relevant details: loaded core, loaded ROM's hash, open tools, running scripts.. -->
+
+### Expected Results
+
+<!-- What you expected to happen. -->
+
+### Actual Results
+
+<!-- What actually happened. If an exception occurred or a crash dump was produced, please include it. -->
+
+### Checklist
+
+<!-- Put an x inside [ ] to check the box, like so: [x] -->
+
+- [ ] This issue is related to EmuHawk.
+- [ ] This issue is related to DiscoHawk.
+- [ ] This issue is related to an emulation core.
+- [ ] I have checked that this issue is reproducable on the latest development build (found in the README).
```

Comment:
Hey, I've stolen this idea and updated the base branch. If you have anything more to add (or remove), could you rebase and do those changes instead of replacing the file? Thanks.

Reply:
The actual link to the latest dev build is https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts